### PR TITLE
Av 1054 orientation

### PR DESCRIPTION
**Pull request by jwu246**
temp and pressure finished orientation.cpp and sensor_data.h

### CODE REVIEW

File: MIDAS/src/errors.h [hunk 1]
```diff
@@ -3,5 +3,7 @@
 
 enum ErrorCode {
     NoError,
-    LowGCouldNotBeInitialized
+    LowGCouldNotBeInitialized,
+    CANNOT_CONNECT_BNO,
```
Comment (redindelible) [inline MIDAS/src/errors.h:7]: match error naming style

File: MIDAS/src/hardware/Orientation.cpp [hunk 2]
```diff
@@ -1,17 +1,88 @@
 #include "sensors.h"
 
 // #include sensor library
+#include "Adafruit_BNO08x.h"
 
 // global static instance of the sensor
+Adafruit_BNO08x imu;
+sh2_SensorId_t reportType = SH2_ARVR_STABILIZED_RV;
+long reportIntervalUs = 5000;
```
Comment (redindelible) [inline MIDAS/src/hardware/Orientation.cpp:9]: make these two lines #define's

File: MIDAS/src/hardware/Orientation.cpp [hunk 3]
```diff
@@ -1,17 +1,88 @@
...
+        sensor_reading.gy = event.un.gyroscope.y;
+        sensor_reading.gz = event.un.gyroscope.z;
+
+        sensor_reading.magnetometer.mx = event.un.magneticField.x;
+        sensor_reading.magnetometer.my = event.un.magneticField.y;
+        sensor_reading.magnetometer.mz = event.un.magneticField.z;
+    
+        sensor_reading.temperature = event.un.temperature.value;
+        sensor_reading.pressure = event.un.pressure.value;
+    }
```
Comment (redindelible) [inline MIDAS/src/hardware/Orientation.cpp:86]: you're missing the `return sensor_reading;`

**Review by redindelible — CHANGES_REQUESTED** [review]
Make sure there's also an Orientation sensor thread

**Review by aidancostello — APPROVED** [review]

**Review by redindelible — APPROVED** [review]

**Pull request merged**